.infrastructure/helm-chart/todoapp/charts/mysql/templates/statefulSet.yml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Chart.Name }}-stateful-set
+  namespace: {{ .Values.common.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.mysql.labelName }}
+  replicas: {{ .Values.statefulSet.replicas }}
```

Comment:
Ensure that `.Values.statefulSet.replicas` is defined in your `values.yaml` file to specify the number of replicas for the StatefulSet.